sys/net/include/rpl.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2013, 2014  INRIA.
+ * Copyright (C) 2013 - 2015  INRIA.
```

Comment:
(In my personal opinion, refactoring are actual changes to the code and should be subject to copyright law. But as IANAL, do not take my word for it.)